server/src/db/serializers/DogParksSerializer.js

```diff
@@ -10,6 +10,34 @@ class DogParksSerializer {
         })
         return newParks
     }
-}
+
+    static showDetailsForShow(dogPark) {
```

Comment:
I would rename this so that you dont repeat yourself

detailsForShow works better